weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -328,6 +338,16 @@ MyApplet.prototype = {
    }
 
   // Override Methods: Applet
+, _onKeySettingsUpdated: function _onKeySettingsUpdated() {
+    if (this.keybinding != null)
+      Main.keybindingManager.addHotKey(UUID,
+                                       this.keybinding,
+                                       Lang.bind(this,
+                                                 this.on_applet_clicked))
+
+    this.on_applet_clicked
```

Comment:
You don't need that method there on line 348, it won't do anything. Lang.bind binds ```this``` to the method's context when its invoked from the signal event, so you don't need to call it again.